### PR TITLE
Update abyssal manifest.json

### PR DESCRIPTION
**Pull request by TimotejMC**
## Type of change

- [] Add new directory for a server
- [x] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/abyssal/manifest.json [hunk 1]
```diff
@@ -15,5 +15,8 @@
     },
     "discord": {
         "server_id": "592735008427343873"
+    },
+    "brand": {
+        "primary": "#2BBFF0",
```
Comment (scrummer) [inline minecraft_servers/abyssal/manifest.json:20]: Remove the extra comma to make the tests pass

**Review by scrummer — CHANGES_REQUESTED** [review]
There are some issues with your PR. Please fix them so we can merge it.

BR Scrummer

File: minecraft_servers/abyssal/manifest.json (hunk 1 above)
Comment (TimotejMC) [reply minecraft_servers/abyssal/manifest.json:20]: Ok I'm going to do it.

**Comment by scrummer** [comment]
Now it looks good - thank you very much!

BR Scrummer

**Pull request merged**